### PR TITLE
[REVIEW] Update Build instructions in the README

### PR DESCRIPTION
**Pull request by rnyak**
@rnyak please click the edit button and write a description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ environment created in step 3 is active.
 
 ```
 git clone https://github.com/rapidsai/cuspatial.git $CUSPATIAL_HOME
+cd $CUSPATIAL_HOME
```
Comment (harrism) [inline README.md]: Why not just `git clone --recurse-submodules https://github.com/rapidsai/cuspatial.git $CUSPATIAL_HOME`

?

File: docs/Makefile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```
Comment (harrism) [inline docs/Makefile]: These changes should probably be a separate PR. Distinct from the readme...

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Building Documentation
+
+As a prerequisite, a RAPIDS compatible GPU is required to build the docs since the notebooks in the docs execute the code to generate the HTML output.
```
Comment (harrism) [inline docs/README.md]: Is this true of cuSpatial?

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+1. Create a conda env and build cuspatial from source. The dependencies to build rapids from source are installed in that conda environment, and then rapids is built and installed into the same environment.
+
+2. Once cudf is built from source, navigate to `cuspatial/docs/`. If you have your documentation written and want to turn it into HTML, run makefile:
+
+
+```bash
+#be in the same directory as your Makefile
+make html
+```
+This should run Sphinx in your shell, and outputs to `build/html/index.html`.
```
Comment (harrism) [inline docs/README.md]:
```suggestion
This runs Sphinx in your shell, and outputs to `build/html/index.html`.
```

File: docs/README.md (hunk 3 above)
Comment (rnyak) [reply docs/README.md]: @thomcom fyi.

**Comment by thomcom** [comment]
Thanks @rnyak. When you get the chance do you mind merging `branch-0.13` into this so that the warnings will not be treated as errors?

File: README.md (hunk 1 above)
Comment (thomcom) [reply README.md]: Agree, `--recurse-submodules` is used in all the cudf scripts as well.

Comment (rnyak) [reply README.md]: Ok.
